recOrder/plugin/widget/main_widget.py

```diff
@@ -2112,6 +2112,17 @@ def enter_acq_bg_path(self):
         else:
             self.ui.le_bg_path.setText('Path Does Not Exist')
 
+    @Slot(str)
+    def handle_bg_path_update(self,value):
+        #Path from Background Worker
+        path = value
+        if os.path.exists(path):
+            self.acq_bg_directory = path
+            self.current_bg_path = path
+            self.ui.le_bg_path.setText(path)
+        else:
+            self.ui.le_bg_path.setText('Path Does Not Exist')
```

Comment:
Refer to: https://github.com/mehta-lab/recOrder/blob/f4d6407ca7d226ffddf04bfc77afc31d677c3aa5/recOrder/plugin/widget/main_widget.py#L2276-L2277

Reply:
Ideally we also want a napari-level warning banner for all these events. Maybe worth a separate issue for 1.0.0+.

Reply:
On second thought, do we need this if the parameter is only passed in from within `recOrder`? How about a try/except instead?